src/corebluetooth.cs

```diff
@@ -148,6 +148,65 @@ interface CBCentralManager {
 #endif
 	}
 
+	[StrongDictionary ("AdvertisementDataKeys")]
+	interface AdvertisementData {
+		string LocalName { get; set; }
+		NSData ManufacturerData { get; set; }
+		NSDictionary ServiceData { get; set; }
```

Comment:
This can probably be a generic dictionary:

"The keys are CBUUID objects, representing CBService UUIDs. The values are NSData objects, representing service-specific data."